packages/unimodules-task-manager-interface/android/src/main/java/org/unimodules/interfaces/taskManager/TaskManagerInterface.java

```diff
@@ -5,6 +5,10 @@
 import java.util.Map;
 
 public interface TaskManagerInterface {
+
+  String EVENT_NAME = "TaskManager.executeTask";
+  String E_TASK_SERVICE_NOT_FOUND = "ERR_TASK_SERVICE_NOT_FOUND";
```

Comment:
```suggestion
  String ERR_TASK_SERVICE_NOT_FOUND = "ERR_TASK_SERVICE_NOT_FOUND";
```

I think we can change that name so it matches its value.